Demo/Application/Base/Base View Controllers/BraintreeDemoPaymentButtonBaseViewController.m

```diff
@@ -23,7 +23,8 @@ - (void)viewDidLoad {
     [self.view addSubview:self.paymentButton];
 
     [NSLayoutConstraint activateConstraints:@[
-        [self.paymentButton.centerXAnchor constraintEqualToAnchor:self.view.centerXAnchor],
+        [self.paymentButton.leadingAnchor constraintEqualToAnchor:self.view.leadingAnchor],
+        [self.paymentButton.trailingAnchor constraintEqualToAnchor:self.view.trailingAnchor],
```

Comment:
Not having a leading and trailing anchor means that the text will expand forever. In the case of printing device data, we don't want to happen. Previously `BraintreeDemoBTDataCollectorViewController` subclassed `BraintreeDemoBaseViewController` so this was a non issue. That doesn't make a ton of sense to have some ViewControllers subclass one view and others subclass a different one so updated for consistency.